### PR TITLE
Feature/trigga save post nr du klickar p spara p spara moduler cu 8693v3uxu

### PR DESCRIPTION
**Pull request by annalinneajohansson**

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-8693v3uxu Trigga save_post när du klickar på "Spara" på Spara moduler](https://app.clickup.com/t/8693v3uxu)

File: phpunit.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
   <testsuites>
     <testsuite name="default">
-      <directory suffix=".php">./tests/phpunit/tests</directory>
+      <directory suffix=".test.php">source/php</directory>
```
Comment (NiclasNorin) [inline phpunit.xml:27]: This should not be changed. Tests are currently placed outside of the source folder. Check here:
https://github.com/helsingborg-stad/Municipio/tree/main/tests/phpunit

File: source/php/BlockManager.php [hunk 2]
```diff
@@ -236,11 +236,12 @@ public function registerBlocks()
                     );
 
                     if (!acf_register_block_type($blockSettings)) {
-                        throw new \WP_Error(
+                        $error = new \WP_Error(
                             'block_editor_create_module',
                             "Could not create block for with the id of " . $class->moduleSlug
                         );
-                    };
+                        error_log($error->get_error_message());
```
Comment (thorbrink) [inline source/php/BlockManager.php:243]: Remove WP_Error and pass error message to error_log directly.

File: source/php/Editor.php [hunk 3]
```diff
@@ -591,42 +595,38 @@ public function save()
...
+            update_post_meta($postId, 'modularity-modules', $data);
         } else {
-            delete_post_meta($key, 'modularity-modules');
+            delete_post_meta($postId, 'modularity-modules');
         }
 
         // Save/remove sidebar options
         if (isset($_POST['modularity_sidebar_options'])) {
-            update_post_meta($key, 'modularity-sidebar-options', $_POST['modularity_sidebar_options']);
+            update_post_meta($postId, 'modularity-sidebar-options', $_POST['modularity_sidebar_options']);
```
Comment (thorbrink) [inline source/php/Editor.php:624]: Sanitize POST data before use.

**Review by thorbrink — CHANGES_REQUESTED** [review]

File: source/php/Editor.php [hunk 4]
```diff
@@ -591,42 +595,38 @@ public function save()
...
         } else {
-            delete_post_meta($key, 'modularity-sidebar-options');
+            delete_post_meta($postId, 'modularity-sidebar-options');
```
Comment (sebastianthulin) [inline source/php/Editor.php:626]: Sanitize need

**Review by sebastianthulin — APPROVED** [review]

**Pull request merged**